configuration/driver/icedrv_step.F90

```diff
@@ -111,9 +111,10 @@ subroutine step_therm1 (dt)
       use icedrv_arrays_column, only: fswsfcn, fswintn, Sswabsn, Iswabsn
       use icedrv_arrays_column, only: fswthrun, fswthrun_vdr, fswthrun_vdf, fswthrun_idr, fswthrun_idf
       use icedrv_arrays_column, only: meltsliqn, meltsliq
+      use icedrv_arrays_column, only: floe_rad_c, floe_binwidth
       use icedrv_calendar, only: yday
-      use icedrv_domain_size, only: ncat, nilyr, nslyr, n_aero, n_iso, nx
-      use icedrv_flux, only: frzmlt, sst, Tf, strocnxT, strocnyT, rside, fside, wlat, &
```

Comment:
It looks like fside is no longer used at all.  Are there any references to it in the docs?

Reply:
I think the docs are OK, we'll have to update the interface documentation though, but that is done through automation.